proto/Networking/Platform/Responses/ListLoginActionResponse.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package POGOProtos.Networking.Responses;
```

Comment:
I don't think so. The platform version is in PlatformAction file

Reply:
J'ai supprimer ListLoginActionResponse.proto je l'ai rajouter dans PlatformClientActions et dans Platform.Requests

Reply:
oui mais non il est aussi dans RequestType

Reply:
ok faut le enlever PlatformClientActions (M/R) .proto comme c'était avant si j'ai bien compris

Reply:
ah non attends j'ai compris, il faut une fois dans Platform Action et une fois dans le repertoire Platform.

Reply:
du coup je pense qu'on est bon

Reply:
fixed

Reply:
a merde je revert alors ???

Reply:
as u want

Reply:
je crois que dans PlatformClientActions (M/R) .proto comme je avait mis au départ était plus propre mais c'est toi le spécialiste

Reply:
ok ben faisons ca

Reply:
ok je revert

Reply:
voila ready